src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/helpers/PremiumAdLayout.java

```diff
@@ -0,0 +1,12 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.helpers;
+
+import com.wikia.webdriver.common.core.url.Page;
+import com.wikia.webdriver.common.core.url.UrlBuilder;
+
+public class PremiumAdLayout {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1118)